common/webSocket/webSocketEvents.js

```diff
@@ -13,4 +13,5 @@ export const WebSocketEvents = {
   recordDelete: 'recordDelete',
   error: 'threadError',
   recordSessionExpired: 'recordSessionExpired',
+  applicationError: 'applicationError',
```

Comment:
since it's an error managed only during record update, can it be something like recordUpdateError?

Reply:
It could be, but I was wondering if there would be use for a generic non-fatal error message type

Reply:
sounds good